packages/website/src/entities/branding/ui/logo.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { MemoryRouter } from "react-router-dom";
+import Logo from "./logo";
+
+test("clicking on the logo redirects to the home page", () => {
+  render(
+    <MemoryRouter>
+      <Logo />
+    </MemoryRouter>,
+  );
+
+  const logo = screen.getByTestId("logo");
```

Comment:
nit. Try to use accessibility roles first before defining and utilizing data test IDs.

Reply:
Thanks for the feedback, @glenflorendo . I deleted the data test id and replaced it with `screen.getByAltText`.